virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc

```diff
@@ -41,17 +44,21 @@ Optionally, you can create a pull secret from your Docker credentials and add it
 
 include::modules/configmap-adding-ca.adoc[leveloffset=+3]
 include::modules/images-allow-pods-to-reference-images-from-secure-registries.adoc[leveloffset=+3]
-
 include::modules/virt-creating-vddk-image.adoc[leveloffset=+1]
-include::modules/virt-importing-vmware-vm.adoc[leveloffset=+1]
 
-See xref:virt-importing-vmware-vm-wizard-fields_virt-importing-vmware-vm[virtual machine wizard fields] for more information on the wizard fields.
+include::modules/virt-importing-vm.adoc[leveloffset=+1]
+
+See xref:../../../virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc#virt-importing-vmware-vm-wizard-fields_virt-importing-vmware-vm[virtual machine wizard fields] for more information on the wizard fields.
+
+virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
```

Comment:
I think this line must be here by mistake.

Reply:
Yup. Paste error